org.eclipse.xtext.testing/src/org/eclipse/xtext/testing/smoketest/internal/RunnerBuilder.java

```diff
@@ -72,6 +72,7 @@ public Runner runnerForClass(Class<?> testClass) throws Throwable {
 		private final ProcessedBy processor;
 		private final Scenario[] scenarios;
 		
+		@SuppressWarnings("deprecation")
```

Comment:
?

Reply:
why was this not deprecated before?

Reply:
is there a junit 4.13 thing that is pulled now?!?

Reply:
Let's schedule the upgrade to 4.13. I think @kthoms started to work on that

Reply:
join https://github.com/eclipse/xtext/issues/1645